src/unit_tests/wazuh_db/test_wdb.c

```diff
@@ -478,7 +478,7 @@ void test_wdb_init_stmt_in_cache_invalid_statement(void **state) {
     will_return(__wrap_sqlite3_finalize, SQLITE_OK);
 
     // wdb_stmt_cache
-    expect_string(__wrap__merror, formatted_msg, "DB(000) SQL statement index (172) out of bounds");
+    expect_string(__wrap__merror, formatted_msg, "DB(000) SQL statement index (174) out of bounds");
```

Comment:
This test will fail every time **WDB_STMT_SIZE** changes. The expected error message should use the constant value and not a fixed number (like 174)

Reply:
Totally right!

Reply:
Fixed